ironfish-cli/src/commands/chain/download.ts

```diff
@@ -86,10 +86,12 @@ export default class Download extends IronfishCommand {
       }
 
       const fileSize = FileUtils.formatFileSize(manifest.file_size)
+      const spaceRequired = FileUtils.formatFileSize(manifest.file_size * 2)
 
       if (!flags.confirm) {
         const confirm = await CliUx.ux.confirm(
-          `Download ${fileSize} snapshot to update from block ${node.chain.head.sequence} to ${manifest.block_sequence}?` +
+          `Download ${fileSize} snapshot to update from block ${node.chain.head.sequence} to ${manifest.block_sequence}? ` +
+            `At least ${spaceRequired} of free disk space is required to download and unzip the snapshot file.` +
```

Comment:
nit: maybe add a new line for `At least ...`

Reply:
updated!